### ISSUE TITLE
Migrate Yale materials from YIPP

### ISSUE THREAD

**Issue by sshetenhelm** [yalelibrary/YUL-DC#2744]
**Story**
Migrate only the Yale materials from the Yale Indian Papers Project (New England Indian Papers Series) Collection (**PID 27**) from FindIt into DCS. 

Migrate only the parents in the following CSVs:
BRBL repo (80 parents) -- 

MSSA repo (356 parents) -- 

**Acceptance**
- [ ]  Export image files from Fedora
- [ ]  Import image files into the DCS pairtree
- [ ]  Generate list of Parent OIDs [use OID lists above]
- [ ]  Import the BRBL objects into DCS UAT into the BRBL admin set
- [ ]  Import the MSSA objects into DCS UAT into the MSSA admin set
- [ ] Cross-reference objects with MADID collection and remove duplicates
- [ ] Audit objects in DCS UAT with collection owner
- [ ]  Import the BRBL objects into DCS PROD into the BRBL admin set
- [ ]  Import the MSSA objects into DCS PROD into the MSSA admin set
- [ ]  Audit objects in DCS PROD
- [ ]  Update handles URLs to point to DCS

**Comment by MaggieZhaoYale** [comment]
@sshetenhelm 1-348 parent oid in YIPP MSSA are inactive in findit in prod. I want to double-check with you.

**Comment by sshetenhelm** [comment]
@MaggieZhaoYale Is there an easy way for you to export which OIDs are inactive? If not, I can check.

**Comment by MaggieZhaoYale** [comment]
346 MSSA are not found in Findit


[YIPP_MSSA_not_find_findit.csv](https://github.com/yalelibrary/YUL-DC/files/14422940/YIPP_MSSA_not_find_findit.csv)
5 BRBL are not found in Findit:
10681643
10681717
10682904
10682918
10682919
No full text found for BRBL/MSSA. 

Processed the parents found in Findit in DCS UAT:
https://collections-uat.library.yale.edu/management/batch_processes/1830
https://collections-uat.library.yale.edu/management/batch_processes/1829

**Comment by sshetenhelm** [comment]
Thank you @MaggieZhaoYale , I'll look into these.

**Comment by sshetenhelm** [comment]
Yep, if the information in LB / FindIt is accurate and uniform, then there are only 75 published BRBL and 10 published MSSA objects in YIPP. 

@MaggieZhaoYale  could you please try to bring over MSSA OID **10682343**? This did not get ingested but is public in FindIt. 

I've removed the 5 inactive BRBL parents and 1 MSSA parent that did successfully migrate, but is **not** available in FindIt under YIPP (it has, however, been uploaded to DCS PROD, I'm assuming with MADID).

**Comment by MaggieZhaoYale** [comment]
@sshetenhelm Added the one mssa to UAT https://collections-uat.library.yale.edu/management/batch_processes/1839/parent_objects/10682343

**Comment by sshetenhelm** [comment]
Thanks @MaggieZhaoYale 

We have approval to move into PROD :)

**Comment by MaggieZhaoYale** [comment]
mssa:https://collections.library.yale.edu/management/batch_processes/16042
brlb: https://collections.library.yale.edu/management/batch_processes/16041

**Comment by sshetenhelm** [comment]
Looks great on PROD and handles updated!

Thank you so much, @MaggieZhaoYale 

Ready to close after we point :)
